Fix UG and DG inconsistencies, grammar issues and minor bug fix (#323)
AY2223S1-CS2103T-T12-1/tp: docs/UserGuide.md

  * e.g `n/NAME [c/CLASS_NAME]` can be used as `n/John Doe c/CS1231S` or as `n/John Doe`.
* Items with `...` after them can be used multiple times.
  * e.g. `[c/CLASS_NAME...]` can be used as ` ` (i.e. 0 times since it is also optional), `c/CS1101S`, `c/CS2030 c/ST2334` etc.

> * Parameters, excluding the index parameter, can be in any order.

Reviewer: ```suggestion
* Parameters, excluding index parameters, can be in any order.
```

  * e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
  * e.g. while `assign INDEX c/CLASS_NAME` is acceptable, `assign c/CLASS_NAME INDEX` is not acceptable.

> * If a parameter is expected only once in the command but you specified it multiple times, the parser takes the last occurrence of the parameter only.

Reviewer: ```suggestion
* If a parameter is expected only once in a command but you specified it multiple times, the parser takes only the last occurrence of the parameter.
```

>   * e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` are being taken.

Reviewer: ```suggestion
  * e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
```

* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) are ignored by the parser.

>   * e.g. if the command specifies `help 123`, the parser interprets as `help`.

Reviewer: ```suggestion
  * e.g. if the command specifies `help 123`, the parser interprets it as just `help`.
```

| `focus`    | `focus c/CLASS_NAME`                                               |
| `clear`    | `clear`                                                            |

### View help : `help`

{% include note.html content="

Redirects you to this User Guide page. If the attempt was unsuccessful, the following help window appears.
" %}

![help message](images/helpMessage.png)

> The above message emerges when the redirection failed.

Reviewer: ```suggestion
The above message emerges when the redirection fails.
```

Format: `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS]`
* Edits student data at the specified `INDEX`.

> * Only fields that are specified are being modified.

Reviewer: ```suggestion
* Only specified fields are modified.
```

" %}

Format: `list`

> * The list of students you see contains all the students in TA-Assist.

Reviewer: ```suggestion
* The displayed list of students contains all the students in TA-Assist.
```

" %}

Format: `adds s/SESSION_NAME... [d/DATE]`

> * Creates new sessions with name `SESSION_NAME` on the same `DATE`. If the `DATE` field is empty, the current date is being used.

Reviewer: ```suggestion
* Creates new sessions with names `SESSION_NAME` on the same `DATE`. If the `DATE` field is empty, the current date is used instead.
```

* `DATE` field should follow the format `YYYY-MM-DD`.
* The session names are **case-insensitive** (i.e. if a session **Lab 1** already exists, `adds s/lab 1` does not create a new session **lab 1**). 

Example:

> - `adds s/Lab1 s/Tutorial1 d/2022-08-11` creates sessions `Lab1` and `Tutorial1` on 11  August 2022.

Reviewer: ```suggestion
- `adds s/Lab1 s/Tutorial1 d/2022-08-11` creates sessions `Lab1` and `Tutorial1` on 11 August 2022.
```
